src/parse-tsconfig/index.ts

```diff
@@ -1,10 +1,10 @@
 import fs from 'fs';
 import path from 'path';
-import { parse } from 'jsonc-parser';
-import slash from 'slash';
 import type { TsConfigJson, TsConfigJsonResolved } from '../types';
 import { normalizePath } from '../utils/normalize-path';
 import { resolveExtends } from './resolve-extends';
+import slash from 'slash';
+import { parse } from 'jsonc-parser';
```

Comment:
Is this necessary?

Reply:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/8336744/178635982-78d0240f-7008-41a2-aca1-b85c56a58461.png">

Seems moved by prettier or something else?

Reply:
I need to do a rebase soo.

Reply:
Seems modified by eslint `import/order` rule.

Reply:
Running `npm run lint` doesn't give me any `import/order` errors.

Are you running it with a different config?

Reply:
> Running `npm run lint` doesn't give me any `import/order` errors.
> 
> Are you running it with a different config?

Did you try on this PR?

Reply:
Yep. Please revert all unnecessary changes.
<img width="953" alt="Screen Shot 2022-07-13 at 9 46 11 AM" src="https://user-images.githubusercontent.com/1075694/178748961-06b5e579-980e-4527-867d-4baf591c6595.png">

Reply:
I suppose you didn't use `yarn` to install deps?